src/node.cc

```diff
@@ -127,6 +127,18 @@ typedef int mode_t;
 extern char **environ;
 #endif
 
+/*
+ * This is used to load built-in modules. Instead of using
+ * __attribute__((constructor)), we call the _register_<modname>
+ * function for each built-in modules explicitly in
+ * node::RegisterBuiltinModules(). This is only forward declaration.
+ * The definitions are in each module's implementation when calling
+ * the NODE_MODULE_CONTEXT_AWARE_BUILTIN.
+ */
+#define DECLAREFUN(modname, impl) void _register_##modname(void) impl
```

Comment:
Tiny nit: `void _register_##modname()` (no void argument list, that's a C-ism) and perhaps you can call the macro simply V?  Also applies to line 4999.